test_scripts/SDL5_0/Transfer_RPC_with_invalid_image/003_Alert.lua

```diff
@@ -62,7 +62,11 @@ local requestParams = {
     }
   },
   playTone = true,
-  progressIndicator = true
+  progressIndicator = true,
+	alertIcon = {
```

Comment:
Inconsistent indentation

Reply:
updated in [348bfec](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2208/commits/348bfec2c6f69498804b020063b70380cba08f36)